.github/workflows/autotag.yml

```
@@ -0,0 +1,30 @@
+name: Auto Tag
+
+on:
+  schedule:
+    - cron: "1 12 * * 5"
+
+jobs:
+  tag:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Get Latest Tag
+        id: current_tag
+        run: echo "tag=$(git describe --tags --abbrev=0)" >> $GITHUB_OUTPUT
+
+      - name: Create tag
+        id: create_tag
+        run: echo "tag=${{ steps.current_tag.outputs.tag }}-${{ github.run_number }}" >> $GITHUB_OUTPUT
```

Maybe we could simplify? 

```suggestion
      - name: Create tag
        id: create_tag
        run: echo "tag=$(git describe --tags --abbrev=0)-${{ github.run_number }}" >> $GITHUB_OUTPUT
```

---

True, doesn't serve much purpose to split it